README.md

```diff
@@ -1 +1,107 @@
-# check-handler-plugin
\ No newline at end of file
+# Check Plugin Template
+
+## Overview
+check-plugin-template is a template repository which wraps the [Sensu Plugin SDK][2].
+To use this project as a template, click the "Use this template" button from the main project page.
+Once the repository is created from this template, you can use the [Sensu Plugin Tool][9] to
+populate the templated fields with the proper values.
+
+## Functionality
+
+After successfully creating a project from this template, update the `checkOptions` struct in
+addition to the `executeCheck` and `checkArgs` functions in [main.go][7] to customize the behavior
+of this check plugin.
```

Comment:
It looks like one also has to update the `options` slice.

Reply:
`options` references `checkOptions` so I'm not sure that it needs to be updated.

Reply:
Sorry, maybe I didn't explain well enough; if one only updates [`checkOptions`](https://github.com/sensu-community/check-plugin-template/blob/9ff0e7684258f2e3a1c8fb20213402a176f0bd1d/main.go#L31-L41), it will either not pick up their new options or result in a compilation error if they remove the "Example" option.

One has to also update [the `options` slice](https://github.com/sensu-community/check-plugin-template/blob/9ff0e7684258f2e3a1c8fb20213402a176f0bd1d/main.go#L43-L46) and [the `Options` type](https://github.com/sensu-community/check-plugin-template/blob/9ff0e7684258f2e3a1c8fb20213402a176f0bd1d/main.go#L18-L20).